openrouteservice/src/main/java/heigit/ors/api/requests/routing/RouteRequestHandler.java

```diff
@@ -99,6 +99,25 @@ public  RoutingRequest convertRouteRequest(RouteRequest request) throws StatusCo
             }
         }
 
+        if (request.hasSkipSegments()) {
+            String skipSegmentsName = "skip_segments";
+            routingRequest.setSkipSegments(request.getSkipSegments());
+            for (int i = 0; i < request.getSkipSegments().size(); i++) {
```

Comment:
This would be cleaner in a method that is called by the convertRouteRequest method